src/Layer/Launchpad/Launchpad.lua

```diff
@@ -118,9 +118,30 @@ function Launchpad:send(channel, number, value)
     if (not self.midi_out or not self.midi_out.is_open) then
         return
     end
-    local message = {channel, number, value}
-    -- print(("Launchpad : send MIDI %X %X %X"):format(message[1], message[2], message[3]))
-    self.midi_out:send(message)
+	if (channel == 0x90 and value > 127) then
```

Comment:
this is for flash buttons. flash is handled with sysex in lp pro, so bit 7
is used for signaling flash, while bits 0-6 are the color code.

Il domenica 28 febbraio 2016, Ingolf Wagner <notifications@github.com> ha
scritto:

> In src/Layer/Launchpad/Launchpad.lua
> <https://github.com/mrVanDalo/stepp0r/pull/90#discussion_r54348576>:
>
> > @@ -118,9 +118,30 @@ function Launchpad:send(channel, number, value)
> >      if (not self.midi_out or not self.midi_out.is_open) then
> >          return
> >      end
> > -    local message = {channel, number, value}
> > -    -- print(("Launchpad : send MIDI %X %X %X"):format(message[1], message[2], message[3]))
> > -    self.midi_out:send(message)
> > +	if (channel == 0x90 and value > 127) then
>
> what is this for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrVanDalo/stepp0r/pull/90/files#r54348576>.
>